documentation/Thermal simulations getting started.md

```diff
@@ -2,6 +2,19 @@
 - The thermal simulations themselves are based on the output of the acoustic simulations.
 	- So running the thermal simulations without any acoustic output to base it on is not an option.
 
+Before feeding in parameters for the heating simulations it is important to note that these simulations don't model the individual pulses and their pulse-repetition frequencies (PRF) but concatenate the pulses over a trail into one block of stimulation and one block without.
+
+# Example
+Let's take a PRF of 5Hz, a stim_duration of 1s, an iti of 15s, 30 trials and a duty_cycle of 30% (0.3).
+When running the stimulation, these are the pressure changes the transducer will produce.
+- 40ms of stimulation, 160ms without stimulation.
+- This cycle is repeated 5 times (5Hz) within your stim_duration of 1s
```

Comment:
and PRF doesn't matter as you write above, so it is just a coincidence that it repeats 5 times

Reply:
How is the PRF incorporated then?

Reply:
I updated this file in the main branch, could you please take a look and see if anything else needs to be changed?